tests/manager/manager-executeQuery-001.phpt

```diff
@@ -71,7 +71,7 @@ object(MongoDB\Driver\Cursor)#%d (%d) {
   ["readPreference"]=>
   NULL
   ["isDead"]=>
-  bool(true)
+  bool(false)
```

Comment:
Per https://github.com/mongodb/mongo-php-driver/pull/322/files#r63595485, the cursor is no longer dead because we've yet to iterate to any of the documents in the single-batch reply for this query.